src/webui/api/serialize/serialize_torrent.h

```diff
@@ -69,6 +69,7 @@ inline const QString KEY_TORRENT_CONTENT_PATH = u"content_path"_s;
 inline const QString KEY_TORRENT_ADDED_ON = u"added_on"_s;
 inline const QString KEY_TORRENT_COMPLETION_ON = u"completion_on"_s;
 inline const QString KEY_TORRENT_TRACKER = u"tracker"_s;
+inline const QString KEY_TORRENT_TRACKERS = u"trackers"_s;
```

Comment:
I wasn't certain if this file was the best place for these new keys.

Reply:
Moved to a new section in `torrentscontroller.cpp`.

Reply:
> Moved to a new section in `torrentscontroller.cpp`.

I would put it into `serialize/serialize_torrentinfo.*` as `serialize(const TorrentInfo &)`.

Reply:
This becomes tricky as it reuses keys defined in `torrentscontroller.cpp`, and I don't think it'd be worthwhile to duplicate them. I'm going to leave the serialize function in the anonymous namespace in `torrentscontroller.cpp` since that pattern is heavily used.